apps/explorer/lib/explorer/celo/validator_group_rewards.ex

```diff
@@ -56,26 +57,33 @@ defmodule Explorer.Celo.ValidatorGroupRewards do
       |> CeloContractEvent.query_by_group_param(group_address_hash)
       |> Repo.all()
 
-    case activated_votes_for_group do
-      [] ->
-        {:error, :not_found}
+    structured_activated_votes_for_group =
```

Comment:
This is essentially duplicated in [apps/explorer/lib/explorer/celo/validator_rewards.ex](https://github.com/celo-org/blockscout/pull/590/files#diff-cee71614780948baf61273a066bdad92176682c5de23bb72ba3d025056d2b698)

By preventing duplication we can share the same tests, reduce the scope of bugs and make it easier to maintain in future